source/modulo_interfaces/msg/EncodedState.msg

```diff
@@ -0,0 +1,2 @@
+std_msgs/MultiArrayLayout layout
```

Comment:
we don't need this since we only ever read and write the `data` field. The layout is there for flattening / unflattening more complex structures (like a matrix, nested arrays or other kinds of tensors), but we just care about storing our encoded byte string

```suggestion
```